src/betterproto/__init__.py

```diff
@@ -802,7 +803,12 @@ def _postprocess_single(
                 elif meta.wraps:
                     # This is a Google wrapper value message around a single
                     # scalar type.
-                    value = _get_wrapper(meta.wraps)().parse(value).value
+                    wrapper = _get_wrapper(meta.wraps)().parse(value)
+                    if meta.wraps == "enum":
```

Comment:
Shouldn't this use TYPE_ENUM?

Reply:
yea, that would make sense. Feel free to make commits on top of this one if it's useful to do so.

Reply:
I do not have the brains for that lol.